src/task.ts

```diff
@@ -64,8 +62,9 @@ export class TaskManager {
   }
 
   runTask(taskId: number): Promise<void> {
-    if (this.#executorCmd && this.#executorCmd.length > 0) {
-      const cmd = `${this.#executorCmd} --runner-url=ws://localhost:${this.#listeningPort} --task-id=${taskId}`
+    const executorCmd = process.env.EXECUTOR_CMD || ''
```

Comment:
better to pass the value instead of depends on a global variable
all access of process.env should be in the entry point for discovery purpose